api/logging/src/main/java/io/opentelemetry/logging/LogEmitter.java

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.logging;
+
+public interface LogEmitter {}
```

Comment:
We don't want to create a logging API at this point. Only SDK support for implementing log appenders.